lib/svg-sprite/shape.js

```diff
@@ -165,8 +165,8 @@ function SVGShape(file, spriter) {
 
     // Determine meta & alignment data
     const relative = path.basename(this.source.relative, '.svg');
-    this.meta = this.id in this.config.meta ? this.config.meta[this.id] : (relative in this.config.meta ? this.config.meta[relative] : {});
-    this.align = Object.entries({ ...this.config.align['*'], ...(this.id in this.config.align ? this.config.align[this.id] : (relative in this.config.align ? this.config.align[relative] : {})) });
+    this.meta = this.config.meta && this.id in this.config.meta ? this.config.meta[this.id] : (this.config.meta && relative in this.config.meta ? this.config.meta[relative] : {});
```

Comment:
I added these changes because without them it's impossible to cover the file by the tests. The problem is if we do not provide `spriter.config.shape` , the `config.meta` and `config.align` became `undefined`. Because at the line 150 `config` merges with empty object. 
```
this.spriter.config.shape || {}
```

We should either keep this changes or remove this condition to merge with `this.spriter.config.shape` only

Reply:
Unsure how to proceed, I'll leave the choice to you. Just keep in mind that if it's something for tests only, we shouldn't change the code. If it's a real bug (which it seems so), then you can go ahead with a fix.

Reply:
@XhmikosR OK, I'll reverse this code back and we'll just ignore this line for now. Not a big problem. We'll go back to this later, I'll add **TODO**